drivers/ili9341/ili9341.c

```diff
@@ -248,7 +250,7 @@ void ili9341_fill(const ili9341_t *dev, uint16_t x1, uint16_t x2, uint16_t y1,
     _ili9341_set_area(dev, x1, x2, y1, y2);
     /* Memory access command */
     _ili9341_cmd_start(dev, ILI9341_CMD_RAMWR, true);
-#if ILI9341_LE_MODE
+#if IS_ACTIVE(CONFIG_ILI9341_LE_MODE)
```

Comment:
I think here and bellow the `#if` could be converted to `if (IS_ACTIVE(CONFIG_ILI9341_LE_MODE)) {`

Reply:
Done.